package.json

```diff
@@ -13,8 +13,8 @@
     "build": "grunt build",
     "lint": "npm-run-all lint:*",
     "lint:php": "composer lint",
-    "lint:css": "wp-scripts lint-style ./user-edit.css ./providers/css/",
-    "lint:js": "wp-scripts lint-js ./Gruntfile.js ./providers/js/",
+    "lint:css": "wp-scripts lint-style ./user-edit.css", // ./providers/css/",
+    "lint:js": "wp-scripts lint-js ./Gruntfile.js", // ./providers/js/",
```

Comment:
Unfortunately JSON doesn't support comments JS-style like this and will make NPM and other tools fail when attempting to parse it. I feel like we can safely remove the additional folders paths. There is this pull request with the changes clearly illustrated.

Reply:
removed in [b50326f](https://github.com/WordPress/two-factor/pull/439/commits/b50326f3299fc61ccf924f4c3e78434b05fdc1af)